bin/reprocessing_routines.sh

```diff
@@ -112,9 +112,38 @@ get_magetab_for_experiment() {
 
 # 7: see directory, see its contents, can write (fg_atlas only)
 # 5: see directory, see its contents: public read-only directory
+
+## get privacy status for any experiments
+## -MTAB- experiments loaded by biostudies-AE/Annotare uis checked internal file
+## -GEOD-/-ERAD-/-ENAD- are loaded as public from now on
+get_biostudies_privacy_status() {
```

Comment:
Not sure which rule this would be used in, but I saw that in some rules, the `generic_routines.sh` script is sourced along with `reprocessing_routines.sh` (as [here](https://github.com/ebi-gene-expression-group/bulk-recalculations/blob/e4f47498c9d2aaac9f28008c2f749b394d136ede/Snakefile#L1117-L1118)) in which case redefining this function will no longer be needed when [this PR](https://github.com/ebi-gene-expression-group/atlas-prod/pull/254) is merged.

Reply:
The example you mentioned
```
source {workflow.basedir}/atlas-bash-util/generic_routines.sh
```
comes from local submodule `atlas-bash-util` while the PR you mention if from `atlas-prod`

Reply:
Oh sorry, I mis-traced.  LGTM.